packages/seacas/libraries/ioss/src/Ioss_Decomposition.C

```diff
@@ -161,6 +161,128 @@ namespace Ioss {
     return valid_methods;
   }
 
+  size_t get_all_block_ioss_element_size(const std::vector<BlockDecompositionData> &blocks)
+  {
+    size_t count = 0;
+
+    for(size_t blk_seq = 0; blk_seq < blocks.size(); blk_seq++) {
+      const Ioss::BlockDecompositionData& block = blocks[blk_seq];
+
+      // Determine number of file decomp elements are in this block and the offset into the block.
+      count += (block.importMap.size() + block.localMap.size());
+    }
+
+    return count;
+  }
+
+  size_t get_all_block_ioss_offset_size(const std::vector<BlockDecompositionData> &blocks,
```

Comment:
I wanted a quick function to get the max offset value instead of calling `get_all_block_ioss_offset` and looking at the value in the last vector index. I suppose it's a cheap enough function that it shouldn't matter if a vector is temporarily created ... I was just thinking of cases where there is a crazy number of element blocks because the length of the vector is num_blocks+1. Still, we could eliminate this function if necessary

Reply:
I think the function is ok, I just was confused about the `offset` in the name since it is really summing the total number of something `elements * components` or `elements`.  That happens to be the size of the offset value in the offset vector, but I guess I see the `offset` in the name as confusing since it is more an artifact of what the offset is actually representing...